pkg/controller/perconaservermongodb/mgo.go

```diff
@@ -38,6 +38,11 @@ var errReplsetLimit = fmt.Errorf("maximum replset member (%d) count reached", mo
 
 func (r *ReconcilePerconaServerMongoDB) reconcileCluster(cr *api.PerconaServerMongoDB, replset *api.ReplsetSpec,
 	pods corev1.PodList, usersSecret *corev1.Secret, mongosPods []corev1.Pod) (mongoClusterState, error) {
+
+	if cr.Spec.Pause {
```

Comment:
I think situation `cr.Status.Replsets[replset.Name].Size: 0` and `cr.Spec.Pause: flase` is possible.
I would prefer to check `cr.Status.Replsets[replset.Name].Size: 0` check

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/477/commits/2e16879713cda7825b44d1755fdf338f2576c794